algebras/build.sbt

```diff
@@ -8,6 +8,7 @@ val algebra =
       name := "endpoints-algebra",
       libraryDependencies ++= Seq(
         "com.github.tomakehurst" % "wiremock" % "2.6.0" % Test,
+        "com.softwaremill.sttp" %% "core" % "1.3.2" % Test,
```

Comment:
The CI failed with an error related to this dependency. Can you check that we don’t miss another dependency?

Reply:
It seems `sttp` has dependencies to `akka-http 10.1.4` and `akka-stream 2.5.16`, maybe it's the reason of failure?

Reply:
I think changing the version of the `sttp` to `1.1.13` can help.